app/command_channel_rename.go

```diff
@@ -25,12 +25,15 @@ func (me *RenameProvider) GetTrigger() string {
 }
 
 func (me *RenameProvider) GetCommand(a *App, T goi18n.TranslateFunc) *model.Command {
+	renameAutocompleteData := model.NewAutocompleteData(CMD_RENAME, T("api.command_channel_rename.hint"), T("api.command_channel_rename.desc"))
+	renameAutocompleteData.AddTextArgument(T("api.command_channel_rename.hint"), "[text]", "")
```

Comment:
There are also other commands like `code`that need to be updated. Is there a ticket to do that?

Reply:
Yes https://mattermost.atlassian.net/browse/MM-21499. We should make them HW.